components/counter-bar/src/index.js

```diff
@@ -240,7 +257,7 @@ CounterBar.Item = ({
 }) => {
   const Wrapper = wrapper.withComponent(component);
   return (
-    <Wrapper active={active} disabled={!score} empty={ !children || children.length === 0 ? 1 : 0 } {...props}>
+    <Wrapper active={active ? 1 : 0} disabled={!score} empty={ !children || children.length === 0 ? 1 : 0 } {...props}>
```

Comment:
this fix is wrong - as is the use of a pseudo-boolean for `empty` here too - it's a bodge that avoids addressing the underling issue and will instead potentially create other issues with other components we may be using through the `component` prop

the underlying issue here is that we're using emotion to style and emotion by default will pass through props that should be used for styling only on to the underlying component.

what we *should* be doing is explicitly ensuring that we do not pass through props that we're using for styling.

emotion includes support for providing an option `shouldForwardProp` - a function to indicate whether a prop should be forwarded or not which accepts the prop name as a string.  some info on this can be found here: https://github.com/emotion-js/emotion/issues/655

for our purposes on the `TotalWrapper` i believe we'd need something like:
```
shouldForwardProp: prop => ['active', 'empty'].indexOf(prop) === -1
```

Reply:
an example: https://codesandbox.io/s/v3qrkpoj37

Reply:
I see what you mean, Steve, they're unnecessarily being passed on. Will take another look at this on train.

Reply:
Nice one - this issue is a PITA, nice fix!